tests/fixtures/issues/issue-511.php

```diff
@@ -2495,15 +2495,15 @@
     'issue-511-BE' => [
         'ua' => 'Mozilla/5.0 (Android; Mobile; rv:33.1) Gecko/33.1 Firefox/33.1',
         'properties' => [
-            'Comment' => 'Firefox 33.0',
```

Comment:
Since I moved Firefox Android to its own file, I was able to add these ".1" releases (seems there were only two of them so far): https://en.wikipedia.org/wiki/Firefox_for_Android#Release_history

Here's the release announcement for this version: https://www.mozilla.org/en-US/firefox/android/33.1/releasenotes/